main/src/main/java/cgeo/geocaching/filters/gui/FilterViewHolderCreator.java

```diff
@@ -69,7 +69,7 @@ private static IFilterViewHolder<?> createFor(final GeocacheFilterType type, fin
                 result = new CheckboxFilterViewHolder<>(
                         ValueGroupFilterAccessor.<CacheType, TypeGeocacheFilter>createForValueGroupFilter()
                                 .setSelectableValues(Arrays.asList(CacheType.TRADITIONAL, CacheType.MULTI, CacheType.MYSTERY, CacheType.LETTERBOX, CacheType.EVENT,
-                                        CacheType.EARTH, CacheType.CITO, CacheType.WEBCAM, CacheType.VIRTUAL, CacheType.WHERIGO, CacheType.ADVLAB, CacheType.USER_DEFINED))
+                                        CacheType.EARTH, CacheType.CITO, CacheType.WEBCAM, CacheType.SPECIALEVENT, CacheType.VIRTUAL, CacheType.WHERIGO, CacheType.ADVLAB, CacheType.OTHER, CacheType.USER_DEFINED))
```

Comment:
* put here too the "group values" you selected before
* "setValueDisplayTextGetter" and "setValueDrawableGetter" also have to be adjusted for these "group values" to return whatever you want to display in the filter dialog